src/Runtime/LambdaRuntime.php

```diff
@@ -166,6 +166,7 @@ private function postJson(string $url, $data): void
         $handler = curl_init($url);
         curl_setopt($handler, CURLOPT_CUSTOMREQUEST, 'POST');
         curl_setopt($handler, CURLOPT_RETURNTRANSFER, true);
+        curl_setopt($handler, CURLOPT_FAILONERROR, true);
```

Comment:
This constant defaults to true, why do we need it here?

Reply:
Yes it should default to true. But it didn't throw an error when returning with a 4xx status code. So I forced the fail on error flag.

See also https://github.com/mnapoli/bref/blob/master/src/Runtime/LambdaRuntime.php#L75